index.php

```diff
@@ -36,6 +36,9 @@
 
 \Friendica\Core\Logger\Handler\ErrorHandler::register($dice->create(\Psr\Log\LoggerInterface::class));
 
+// Check the database structure and possibly fixes it
+\Friendica\Core\Update::check(\Friendica\DI::basePath(), true, \Friendica\DI::mode());
```

Comment:
We are already having this call here: https://github.com/friendica/friendica/blob/develop/src/App.php#L662

Does this means that it will be called now twice?

Reply:
removed again, you're right!